tests/x11/rrdtool_x11.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE"s openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Regression test for rrdtool
+# - created rrd file
+# - update rrd file
+# - fetch data from file
+# - generate images with different options
+# - compare images with needles
+#
+# Expected result:
+# 4 graphics will be displayed without error.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+my $imagespeed = "";
+
+# open images and check needles.
+sub open_image {
```

Comment:
verify_rrd_image should be better name,as function open and verify image

Reply:
I remove the open_image argument, not make senses use it after review my code.